conans/test/functional/old/conan_trace_file_test.py

```diff
@@ -123,7 +123,7 @@ def test_trace_actions(self):
             self.assertEqual(num_post, 2)  # 2 get urls
 
         num_get = len([it for it in actions if "REST_API_CALL" in it and "GET" in it])
-        self.assertEqual(num_get, 9)
+        self.assertEqual(num_get, 10)
```

Comment:
Why is this?

Reply:
I think this is because in the ``conan user`` the ``authenticate needs to do a ping first, to check if the OATH capability is there. I think that could be a bug in the previous implementation:

```python
if OAUTH_TOKEN in self._cached_capabilities.get(self.remote_url, []):
```
But ``cached_capabilities`` hadn't been populated, because it is not calling to ``_get_api()``